.gitignore

```diff
@@ -3,3 +3,5 @@ spec/dummy
 .bundle
 
 *.log
+
+.project
```

Comment:
I don't know what `.project` file stands for, but `.gitignore` should be used just to ignore files specific to repository content. I think this is not that case and if you want to ignore `.project`, please use [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). Sorry if I'm not getting something.